README.md

```diff
@@ -3,35 +3,13 @@
 
 
 ## Overview
-EdgeX device service for communicating with USB cameras attached to Linux OS platforms.
-This service provides the following capabilities:
-- Camera metadata
-- Camera status
-- Video stream reference
+The USB Device Service is a microservice created to address the lack of standardization and automation of camera discovery and onboarding. EdgeX Foundry is a flexible microservice-based architecture created to promote the interoperability of multiple device interface combinations at the edge. In an EdgeX deployment, the USB Device Service controls and communicates with USB cameras, while EdgeX Foundry presents a standard interface to application developers. With normalized connectivity protocols and a vendor-neutral architecture, EdgeX paired with USB Camera Device Service, simplifies deployment of edge camera devices.
 
-## Tested Devices
-The following devices have been tested with EdgeX:
-<!-- sorted alphabetically -->
-- AUKEY PC-LM1E Webcam
-- HP w200 Webcam
-- Jinpei JW-01B USB FHD Web Computer Camera
-- Logitech Brio 4K
-- Logitech C270 HD Webcam
-- Logitech StreamCam
+Specifically, the device service uses V4L2 API to get camera metadata, FFmpeg framework to capture video frames and stream them to an [RTSP server](https://github.com/aler9/rtsp-simple-server), which is embedded in the docerized device service. This allows the video stream to be integrated into the [larger architecture](#how-it-works).
 
-## How does the device service work?
-- The device service ONLY works on Linux with kernel v5.10 or higher.
-- The device service uses V4L2 API to get camera metadata.
-- The device service uses FFmpeg framework to capture video frames and stream them to an RTSP server.
-- An [RTSP server](https://github.com/aler9/rtsp-simple-server) is embedded in the dockerized device service. 
-
-## General Usage
-
-### Build the executable file
-```shell
-make build
-```
+Use the USB Device Service to streamline and scale your edge camera device deployment. 
 
+## How It Works
```

Comment:
Maybe the build error message should be moved somewhere else? I'm thinking about adding a file dedicated to setting up dependencies, or this could go as a warning in the general-usage doc.